src/screw_execution.cpp

```diff
@@ -0,0 +1,70 @@
+#include <affordance_primitives/screw_execution.hpp>
+#include <affordance_primitives/affordance_utils.hpp>
+#include <affordance_primitives/screw_axis.hpp>
+
+namespace affordance_primitives
+{
+APScrewExecutor::APScrewExecutor() : tfBuffer_(), tfListener_(tfBuffer_)
```

Comment:
I'm not sure if the first `tfBuffer_( )` is needed in this initializer list. 
`tfListener_(tfBuffer_)` works just fine.